agent-instrumentation/src/jvmMain/kotlin/com/epam/drill/agent/instrument/servers/JettyTransformerObject.kt

```diff
@@ -38,11 +39,19 @@ abstract class JettyTransformerObject(
         val agentIdHeader = headersRetriever.agentIdHeader()
         val agentIdValue = headersRetriever.agentIdHeaderValue()
         logger.info { "transform: Starting JettyTransformer with admin host $adminUrl..." }
-        val method =
+        val method = try {
             ctClass.getMethod(
                 "handle",
                 "(Ljava/lang/String;Lorg/eclipse/jetty/server/Request;Ljavax/servlet/http/HttpServletRequest;Ljavax/servlet/http/HttpServletResponse;)V"
             )
+        } catch (e: NotFoundException) {
```

Comment:
Instead of catching exceptions, we can get the existing method using `ctClass.getDeclaredMethods( "handle")` and transform it

Reply:
done